queries/templ/highlights.scm

```diff
@@ -46,8 +46,11 @@
 (element_text) @string.special
 (style_element_text) @string.special
 
+(css_identifier) @keyword
 (css_property
   name: (css_property_name) @attribute)
+(css_property
+  name: (css_property_value) @string)
```

Comment:
This breaks highlighting because you're duplicating a rule for `name`, you need `value`.

Reply:
fixed